app/models/task.py

```diff
@@ -1,6 +1,13 @@
 from flask import current_app
 from app import db
+from sqlalchemy.orm import relationship
+from sqlalchemy import ForeignKey, update
 
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.VARCHAR(50))
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
+    is_complete = db.Column(db.Boolean, default=False)
```

Comment:
I think adding this here creates confusion and is why some of your tests for mark complete/incomplete are failing.  I would remove this line and just add an instance method for self.completed_at. It could be here in the Task model or in routes.py.  

```suggestion
    
```
an example of how that could look
`def build_dict_from_task(task):
    is_complete = True if task.completed_at else False
    dict = {
        "id": task.task_id,
        "title": task.title,
        "description": task.description,
        "is_complete": is_complete
    }`